src/vivarium_census_prl_synth_pop/components/household.py

```diff
@@ -138,65 +139,50 @@ def on_time_step(self, event: Event):
         """
         households = self.population_view.get(event.index)
         household_heads = households.loc[
-            households["relation_to_household_head"] == "Reference person"
-        ]
-        households_that_move = self.addresses.determine_if_moving(
-            household_heads["household_id"], self.household_move_rate
+            households["relation_to_household_head"] == "Reference person"]['household_id']
+
+        households_that_move_idx = self.randomness.filter_for_rate(
+            household_heads.index, self.household_move_rate(household_heads.index)
```

Comment:
Don't we need an additional key for the seed here? I'd recommend doing a Ctrl+F to find all usages of `self.randomness`.

Reply:
Yes.  I have added a key.